spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/ReadIntegrationTestBase.java

```diff
@@ -713,4 +714,42 @@ public void testPushDateTimePredicate() {
             .where("orderDateTime < '2023-10-25 10:00:00'");
     assertThat(df.count()).isEqualTo(2);
   }
+
+  @Test
```

Comment:
Please add some data into the tables, and we need to heave at least one assert. Also, as this refers to ingestion time partitioning, then the partition date should be taken from the test date (dynamically), or to push data into a specific partition using table decorators.